sprocs/check_assessment_access.sql

```diff
@@ -38,7 +38,7 @@ BEGIN
                     END)
             ELSE 'None'
         END AS credit_date_string,
-        aar.time_limit_min,
+        LEAST(aar.time_limit_min, extract(epoch FROM aar.end_date - now()) / 60)::integer AS time_limit_min,
```

Comment:
Existing sprocs aren't totally consistent with capitalization -- not sure of preferred caps for `extract()`, `epoch`, `now()`

Reply:
Our general rule is "use the same formatting rules as the PostgreSQL docs" (I don't think we document this anywhere?). Unfortunately those docs aren't always consistent themselves 😄 

For example: https://www.postgresql.org/docs/12/functions-datetime.html

On that page we see both `interval` and `INTERVAL`. So long as we are not more inconsistent than the postgres docs, I think we are doing ok!